packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx

```diff
@@ -113,6 +110,21 @@ function TenantBasicSettings() {
     }
   };
 
+  useEffect(() => {
+    /**
+     * Redirect to the first tenant if the current tenant is deleted;
+     * Redirect to Cloud console landing page if there is no tenant.
+     */
+    if (tenants && !tenants.some(({ id }) => id === currentTenantId)) {
+      window.open(
```

Comment:
That is not redirect? It will open in a new tab.

Reply:
`window.open` has the default secondary input param as '_blank', which means open a new tab. Here we set it as '_self' which opens a new URL in the current tab. But per offline discussion, use `window.location.assign` instead.